CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+## [3.2.5](https://github.com/segmentio/typewriter/compare/3.2.4...3.2.5) (2018-10-31)
```

Comment:
Maybe we start the release timeline here?

Reply:
(3 major releases in 😂 )

Reply:
Good call! Updated 😄